activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -779,6 +853,27 @@ def add_options_for_index_columns(quoted_columns, **options)
             super
           end
 
+          def exclusion_constraint_name(table_name, **options)
+            options.fetch(:name) do
+              expression = options.fetch(:expression)
+              identifier = "#{table_name}_#{expression}_excl"
+              hashed_identifier = Digest::SHA256.hexdigest(identifier).first(10)
+
+              "excl_rails_#{hashed_identifier}"
+            end
+          end
+
+          def exclusion_constraint_for(table_name, **options)
+            return unless supports_exclusion_constraints?
```

Comment:
This line also is not necessary because it is always true in PostgreSQL adapter.